Python transformations doc update (#1087)
rudderlabs/rudderstack-docs: docs/features/transformations/faq.mdx

## Transformations

#### Is there a limit on the number of transformations I can create and use?

> The RudderStack Cloud **Free** and **Starter** plans let you create 5 JavaScript transformations. However, there is no limit on this number in the **Growth** and **Enterprise** plans. Refer to the <a href="https://www.rudderstack.com/pricing/">Pricing</a> page for more information.

Reviewer: ```suggestion
The RudderStack Cloud **Free** and **Starter** plans let you create 5 JavaScript transformations. However, you can create unlimited transformations in the **Growth** and **Enterprise** plans. Refer to the <a href="https://www.rudderstack.com/pricing/">Pricing</a> page for more information.
```

<div class="infoBlock">

> Python support for creating transformations is only available for the <strong>Growth</strong> and <strong>Enterprise</strong> plans.

Reviewer: ```suggestion
RudderStack supports creating transformations in Python only in the <strong>Growth</strong> and <strong>Enterprise</strong> plans.
```

</div>

#### Are there any sample user transformations I can refer to?

> Yes - refer to the [Sample RudderStack Transformations](https://github.com/rudderlabs/sample-user-transformers) GitHub repository.

Reviewer 2: just `Refer to` would be better
